go.mod

```diff
@@ -268,4 +268,5 @@ require (
 
 go 1.21
 
-replace github.com/hashicorp/hcl/v2 v2.20.1 => github.com/opentofu/hcl/v2 v2.0.0-20240416130056-03228b26f391
+//nolint gomoddirectives // Maintained fork
+replace github.com/hashicorp/hcl/v2 v2.20.1 => github.com/opentofu/hcl/v2 v2.0.0-20240814143621-8048794c5c52
```

Comment:
can we omit using `replace` and change all the imports?

Reply:
It's a large piece of the ecosystem.  Between the massive diff that would be generated and other libraries that require the HC version, it's not a small project.

Long term we need to consider this, but I'm hopeful that we may be able to get changes into the upstream HCL repo.